timescaledb/how-to-guides/hypertables/create.md

```diff
@@ -74,9 +85,79 @@ hypertable fails. For more information about setting up multi-node, see the
      SELECT create_distributed_hypertable('conditions', 'time', 'location');
      ```
 
+<highlight type="note"> 
+If your table has a `PRIMARY KEY` or `UNIQUE` constraint, that key or constraint
+must include all columns that are used to partition your hypertable. In this
+example, the key or constraint must include `time` and `location`.
+
+It may additionally include columns that aren't partitioning columns. For more
+information, see [the troubleshooting
+section](https://docs.timescale.com/timescaledb/latest/how-to-guides/hypertables/create/#primary-keys-and-unique-constraints).
+</highlight>
+
 </procedure>
 
+## Troubleshooting
+
+### Primary keys and unique constraints
+
+If your `PRIMARY KEY` or `UNIQUE` constraint doesn't contain all your
```

Comment:
I'd have this match the copy from above, I don't like the "if it doesn't you get an error", they're probably here because they got that error they want to know something more. 

The other thing about all this copy that I'm still struggling with is that it is rather confusing in terms of what's included in what. I was trying to get at that with the example below - would something like - columns used for partitioning must be included in any `UNIQUE` constraint(s) / `PRIMARY KEY`s on the hypertable be any better? I just find all this very confusing when I'm reading it and I know the topic very well, so it must be confusing for users? How are y'all feeling about it? Do you follow it or find it a bit opaque?

Reply:
➕ I also think it is confusing and I've been struggling with how best to word it. Are the latest edits any clearer?

Reply:
@davidkohn88 friendly bump to this 😃 for when you have a moment